### PR TITLE
Improve code quality

### LINKED ISSUE #20: Consistent style
It could be great to have coding conventions in the README (in Contributing).

### PR DESCRIPTION
**Pull request by pierre-24**
Fixes #20 and #45 (on this repository side).

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 + Bold: `\textbf{text}`
 + Strikethrough: `\sout{text}`
 + Superscript and subscript: `\textsubscript{x}` and `\textsuperscript{x}`
-+ Inline source code: "\verb\`text\`"
++ Inline source code: please use `\texttt{xxx}` rather than `\verb` if possible (see [this post on Stack Exchange](https://tex.stackexchange.com/a/10375))
```
Comment (Karnaj) [inline documentation.md]: Add the fact that special characters should be escaped.

File: zmdocument.cls [hunk 2]
```diff
@@ -218,11 +214,7 @@
 \AtEndDocument{\displaySpoilers
 \printglossary[title=Liste des abbréviations]}
```
Comment (Karnaj) [inline zmdocument.cls]: Style.

```latex
 \AtEndDocument{%
   \displaySpoilers
   \printglossary[title=Liste des abbréviations]
}
```

File: zmdocument.cls [hunk 3]
```diff
@@ -247,8 +239,8 @@
    \setcounter{@authorsNumber}{\the@listSize}
    \setcounter{@columnNumber}{\the\numexpr(1 + \the@authorsNumber / \the@authorsByColumn)\relax}
    \thispagestyle{empty}
-   \pagecolor{bgtitle}
-   \color{texttitle}
+   \pagecolor{titlePageBackgroundColor}
+   \color{titlePageTextColor}
```
Comment (Karnaj) [inline zmdocument.cls]: Set these three last lines in \maketitle macros (they don’t concern authors formatting).

File: zmdocument.cls [hunk 4]
```diff
@@ -247,10 +241,8 @@
    \setcounter{@authorsNumber}{\the@listSize}
    \setcounter{@columnNumber}{\the\numexpr(1 + \the@authorsNumber / \the@authorsByColumn)\relax}
    \thispagestyle{empty}
-   \pagecolor{bgtitle}
-   \color{texttitle}
    \begin{flushright}
-      \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\bsc{##1}}\\}
+      \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\color{internalLinkColor}\bsc{##1}}\\}
```
Comment (Karnaj) [inline zmdocument.cls]: I thought about `\color{texttitle}`. But maybe you should define a `authorLinkColor`.

File: zmdocument.cls [hunk 5]
```diff
@@ -215,14 +211,12 @@
...
 
-\AtEndDocument{\displaySpoilers
-\printglossary[title=Liste des abbréviations]}
-
-%%%
-
+\AtEndDocument{%
+    \displaySpoilers
+    \printglossary[title=Liste des abbréviations]
+}
```
Comment (Karnaj) [inline zmdocument.cls]: 3-space indentation or 4-space?

Comment (pierre-24) [inline zmdocument.cls]: 4-spaces, of course.

Comment (pierre-24) [inline zmdocument.cls]: ... But 3 spaces where used almost everywhere. Therefore, you win :p

Comment (Karnaj) [inline zmdocument.cls]: :)

**Comment by pierre-24** [comment]
Ok, if @Karnaj approves ... :)

**Pull request merged**